src/foraging_gui/Dialogs.py

```diff
@@ -458,6 +458,9 @@ def _LoadCaliPar(self):
 
     def _StartCalibratingLeft(self):
         '''start the calibration loop of left valve'''
+        self.MainWindow._CheckBonsaiConnection()
+        if self.MainWindow.InitializeBonsaiSuccessfully==0:
+            return
```

Comment:
What happens if we return? Does nothing happen? Should the user be alerted in some way?

Reply:
A warning "Please open Bonsai!" will appear in the main window